test/verifiedTests/testFVA/testFVA.m

```diff
@@ -40,8 +41,15 @@
     if solverOK == 1
         fprintf('   Testing flux variability analysis using %s ... ', solverPkgs{k});
 
+        poolobj = gcp('nocreate'); % If no pool, do not create new one.
+        if isempty(poolobj)
+            % launch 2 workers
+            parpool(2);
+        end
+
+            
         % launch the flux variability analysis
-        [minFluxT, maxFluxT] = fluxVariability(model, 90, 'max', model.rxns, 1);
+        [minFluxT, maxFluxT] = fluxVariability(model, 90);
```

Comment:
You could restrict this run to rxnNames and in the following test-loop replace min/maxFlux(i) by min/maxFlux(rxnID(i))
```Matlab
        rxnNames = {'PGI', 'PFK', 'FBP', 'FBA', 'TPI', 'GAPD', 'PGK', 'PGM', 'ENO', 'PYK', 'PPS', 'G6PDH2r', 'PGL', 'GND', 'RPI', 'RPE', 'TKT1', 'TKT2', 'TALA'};

        % launch the flux variability analysis
        fprintf('    Testing flux variability for the following reactions:\n');
        disp(rxnNames);
        [minFluxT, maxFluxT] = fluxVariability(model, 90,'max',rxnNames);


        % retrieve the IDs of each reaction
        rxnID = findRxnIDs(model, rxnNames);

        % check if each flux value corresponds to a pre-calculated value
        for i = 1:size(rxnID)
           % test the components of the minFlux and maxFlux vectors
           assert(minFlux(rxnID(i)) - tol <= minFluxT(i))
           assert(minFluxT(i) <= minFlux(rxnID(i)) + tol)

            assert(maxFlux(rxnID(i)) - tol <= maxFluxT(i))
            assert(maxFluxT(i) <= maxFlux(rxnID(i)) + tol)

            maxMinusMin = maxFlux(rxnID(i)) - minFlux(rxnID(i));
            maxTMinusMinT = maxFluxT(i) - minFluxT(i);
            assert(maxMinusMin - tol <= maxTMinusMinT)
            assert(maxTMinusMinT <= maxMinusMin + tol)

            % print the labels
            printLabeledData(model.rxns(i), [minFlux(i) maxFlux(i) maxFlux(i)-minFlux(i)], true, 3);
        end
```
```